scripts/redcap/wrong_date_associations.py

```diff
@@ -148,7 +150,41 @@ def log_row(row: pd.Series, **kwargs):
     for _, row in errors_df.reset_index().iterrows():
         log_row(row, uid=uid_template, **kwargs)
 
+def get_special_cases(marks, session):
+    """
+    Looks at special_cases.yml file, finds the special cases under the file name section,
+    and then excludes any cases that match the special cases list.
+
+    Note (for Chris use):
+    - 'redcap_event_name' is arm name
```

Comment:
Pedantic: It's *event* name, which also includes arm designation.

Reply:
Gotcha -- thanks for the catch!